lib/redux/actions/utils.js

```diff
@@ -31,6 +31,24 @@ export function getDeviceTargetsByUser(targetsByUser) {
   });
 }
 
+export function getUsersWithTargets(targetDevices, targetTimezones){
```

Comment:
Unit tests for this? Only if it's gonna survive the next steps of persisting to back-end...